source/server-discovery-and-monitoring/tests/unified/interruptInUse-pool-clear.yml

```diff
@@ -5,7 +5,7 @@ schemaVersion: "1.11"
 
 runOnRequirements:
   # failCommand appName requirements
-  - minServerVersion: "4.9"
+  - minServerVersion: "4.4"
```

Comment:
Should this also be 4.4.7?

Reply:
No because the tests in this file don't need to trigger the failpoint on the handshake.